src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/.template.config/localize/templatestrings.de.json

```diff
@@ -4,6 +4,7 @@
   "description": "Eine Projektvorlage zum Erstellen einer Blazor-Web-App, die sowohl serverseitiges Rendering als auch Clientinteraktivität unterstützt. Diese Vorlage kann für Web-Apps mit umfangreichen dynamischen Benutzeroberflächen (UIs) verwendet werden.",
   "symbols/Framework/description": "Das Zielframework für das Projekt.",
   "symbols/Framework/choices/net8.0/description": "Ziel net8.0",
+  "symbols/Framework/choices/net9.0/description": "Ziel net9.0",
```

Comment:
I believe those translations will happen automatically as a separate PR. So you should exclude those changes from your PR.

Reply:
Per the readme, it looked like it was required. Maybe this needs some clarification? 
https://github.com/dotnet/aspnetcore/blob/main/src/ProjectTemplates/README.md#submitting-pull-requests

Per @SteveSandersonMS if the PR is still wanted, I will go back and remove the translation files.

Reply:
> Per @SteveSandersonMS if the PR is still wanted, I will go back and remove the translation files.

Thanks. I wouldn't worry about such details since it would be more useful to use this as a starting point or set of requirements for the bigger redesign effort for .NET 9.

I see that https://github.com/dotnet/aspnetcore/issues/50927 is already tracked in the ".NET 9 planning" milestone so will get considered there, and perhaps could serve as the tracking issue for the proposed redesign. Since that issue already links to this PR, we'll be able to find our way back here (even if this is closed) to see the description and implementation at the right time.